cpp/ql/src/Diagnostics/SuccessfulExtractions.ql

```diff
@@ -0,0 +1,15 @@
+/**
+ * @name Successfully extracted files.
+ * @description Lists all files in the database that were extracted without encountering an error.
```

Comment:
This query includes `exists(f.getRelativePath())`, so it's not all files in the database. Perhaps it's all files in the repo?

Reply:
`s/database/source code directory` as we talk about that term in `codeql database create --help` already.